examples/pylab_examples/axes_props.py

```diff
@@ -11,22 +11,11 @@
 
 t = np.arange(0.0, 2.0, 0.01)
 s = np.sin(2 * np.pi * t)
+
 fig, ax = plt.subplots()
 ax.plot(t, s)
-ax.grid(True)
-
-ticklines = ax.get_xticklines() + ax.get_yticklines()
-gridlines = ax.get_xgridlines() + ax.get_ygridlines()
-ticklabels = ax.get_xticklabels() + ax.get_yticklabels()
-
-for line in ticklines:
-    line.set_linewidth(3)
```

Comment:
As noted in the documentation above, this was actually wrong, since ticks use markers.